lib/TwoFactorAuth.php

```diff
@@ -157,29 +158,23 @@ public function getQRText(string $label, string $secret): string
         return 'otpauth://totp/' . rawurlencode($label)
             . '?secret=' . rawurlencode($secret)
             . '&issuer=' . rawurlencode((string)$this->issuer)
-            . '&period=' . intval($this->period)
+            . '&period=' . $this->period
```

Comment:
good spot on the unnecessary `intval()`'s but should this be a `http_build_query()`?

Reply:
Yeah, I think it can be converted into a `http_build_query`.